### PR TITLE
For #1802596 Deleted XML entities in localized strings.

### PR DESCRIPTION
**Pull request by iorgamgabriel**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### GitHub Automation
Fixes #1802596

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -73,10 +73,14 @@ android {
         beta {
             initWith release
             applicationIdSuffix ".beta"
+            //This is used when the user selects text in other third-party apps. See https://github.com/mozilla-mobile/focus-android/issues/6478
+            manifestPlaceholders = [textSelectionSearchAction: "@string/text_selection_search_action_focus_beta"]
```
[the remark this answers is not in the document]
Comment (iorgamgabriel) [reply app/build.gradle:77]: If the build type is Beta or Nightly, it's the same functionality as before with entities.

File: app/build.gradle [hunk 2]
```diff
@@ -73,10 +73,14 @@ android {
         beta {
             initWith release
             applicationIdSuffix ".beta"
+            //This is used when the user selects text in other third-party apps. See https://github.com/mozilla-mobile/focus-android/issues/6478
```
[the remark this answers is not in the document]
Comment (iorgamgabriel) [reply app/build.gradle]: done

**Review by flodolo — APPROVED** [review]

**Review by jonalmeida — APPROVED** [review]

**Comment by flodolo** [comment]
Could someone merge this, or are the failures actual blockers?

**Comment by iorgamgabriel** [comment]
Thank you @flodolo

**Pull request merged**